Clinical Key.js

```diff
@@ -141,116 +141,95 @@ function doWeb(doc, url){
 		search.getTranslators();
 	}
 	else if (contentType == 'journalArticle') {
-		eid = url.split('/');
-		pii = eid.pop().slice(7);
-		
-		/* can use xpath to get pii; guessing the url is faster
-		var eid = ZU.xpathText(doc, "//ul/@data-eid");
-		pii = eid.slice(7);*/
-		
-		if(!/^S(\d{15}X|\d{16})/.test(pii)){
-			throw new Error('PII incorrectly formatted');
-			return;
-		}
-		
+		try {
+			eid = url.split('/');
+			pii = eid.pop().slice(7);
+			if (!/^S(\d{15}X|\d{16})/.test(pii)){
+				throw new Error('PII from url failed. Trying Xpath');
+			}
+			} catch(e) {
+				Zotero.debug(e);
+				eid = ZU.xpathText(doc, "//ul/@data-eid");
+				pii = eid.slice(7);
+				if (!/^S(\d{15}X|\d{16})/.test(pii)){
+					throw new Error('PII from Xpath failed');
+				}
+			}
 		doSearch(pii, doc);
 	}
 	
 }
 
 //Search & Processing based on CrossRef.js translator
 function doSearch(pii, doc){
-	crossRefQuery = 'http://api.crossref.org/works?query=' + pii
+	crossRefQuery = 'http://api.crossref.org/works?query=' + pii;
+	//acceptHeader = {'Accept': 'application/vnd.crossref-api-message+json; version=1.0'}
 	Zotero.Utilities.HTTP.doGet(crossRefQuery, function(responseText) {
```

Comment:
Just saw @zuphilip's comment on my previous commit. In the next revision, I'll switch this to `ZU.doGet()` as it should work.